ui/lib/kv/addon/components/page/secret/details.js

```diff
@@ -63,6 +69,14 @@ export default class KvSecretDetails extends Component {
     }
   }
 
+  // fetch sync status by destination for kv v2 secret details view
+  @action
+  async fetchSyncStatus() {
+    const { backend, path } = this.args.secret;
+    const syncAdapter = this.store.adapterFor('sync/association');
+    this.syncStatus = await syncAdapter.fetchSyncStatus({ mount: backend, secretName: path });
```

Comment:
This will need to be wrapped in a try/catch. I think silently handling the error is fine for now but might be worth confirming with design.

Reply:
ah thanks! I forgot to move that over when I removed the polling